constants.py

```diff
@@ -105,16 +104,6 @@
     "x86-debian-12-fulltest",
 ]
 
-supportedPlatforms["10.5"] = supportedPlatforms["10.4"].copy()
-
-# Add only 10.4 supported platforms
-supportedPlatforms["10.4"] += [
-    "amd64-kvm-centos-6-bintar",
-    "amd64-kvm-ubuntu-1210-bintar",
-    "x86-kvm-centos-6-bintar",
-    "x86-kvm-ubuntu-1210-bintar",
-]
-
 supportedPlatforms["10.5"] += [
```

Comment:
Can we merge this with the above?

Reply:
Lists merged: https://github.com/MariaDB/buildbot/pull/536/commits/1dcd1739ca0e48a3646d090924ea64e2967590f3